R/pkg/R/functions.R

```diff
@@ -1957,8 +1958,11 @@ setMethod("levenshtein", signature(y = "Column"),
           })
 
 #' @details
-#' \code{months_between}: Returns number of months between dates \code{y} and \code{x}.
 #'
+#' \code{months_between}: Returns number of months between dates \code{y} and \code{x}. 
+#' If \code{y} is later than \code{x}, then the result is positive. If \code{y} and \code{x}
+#' are on the same day of month, or both are the last day of month, time of day will be ignored.
+#' Otherwise, the difference is calculated based on 31 days per month, and rounded to 8 digits.
 #' @rdname column_datetime_diff_functions
```

Comment:
https://github.com/apache/spark/pull/20787#discussion_r177662053 roxygen is very particular with "empty line" which is significant. please change this to 
```
#' @details
#' \code{months_between}: Returns number of months between dates \code{y} and \code{x}. 
#' If \code{y} is later than \code{x}, then the result is positive. If \code{y} and \code{x}
#' are on the same day of month, or both are the last day of month, time of day will be ignored.
#' Otherwise, the difference is calculated based on 31 days per month, and rounded to 8 digits.
#'
#' @rdname column_datetime_diff_functions
```